SpatialGDK/Source/SpatialGDK/Private/Interop/CrossServerRPCHandler.cpp

```diff
@@ -10,6 +10,8 @@ DEFINE_LOG_CATEGORY(LogCrossServerRPCHandler);
 
 namespace SpatialGDK
 {
+static constexpr double CrossServerRPCGuidTimeout = 30.f;
```

Comment:
Better still would be to put it in the class if we don't need it elsewhere. But this is still better.

Reply:
hmm, not sure if I understand. Like `CrossServerRPCHandler::CrossServerRPCGuidTimeout = 30.f`? And make it private?